image/templates/helm/shared/templates/02-scanner-v4-08-matcher-service.yaml

```diff
@@ -0,0 +1,48 @@
+{{- include "srox.init" . -}}
+{{- if ._rox.scannerV4._matcherEnabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: scanner-v4-matcher
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "srox.labels" (list . "service" "scanner-v4-matcher") | nindent 4 }}
+  annotations:
+    {{- include "srox.annotations" (list . "service" "scanner-v4-matcher") | nindent 4 }}
+spec:
+  selector:
+    app: scanner-v4-matcher
+  clusterIP: None
+  ports:
+  - name: grpcs-scanner-v4-matcher
+    port: 8443
+    targetPort: 8443
```

Comment:
`grpc`?

Reply:
should we target `grpc` here since we use the name in the monitoring one below?

Reply:
thanks, done